systemd/systemd.go

```diff
@@ -295,9 +295,9 @@ type Systemd interface {
 	// only necessary to apply manager's configuration like
 	// watchdog.
 	DaemonReexec() error
-	// Enable the given services.
+	// Enable the given services, do not reload systemd.
```

Comment:
it seems that renaming these to EnableNoReload/DisableNoReload would make it easier not to forget to call reload as needed?